test/cri-containerd/layer_integrity_test.go

```diff
@@ -0,0 +1,93 @@
+// +build functional
+
+package cri_containerd
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/Microsoft/hcsshim/internal/oci"
+)
+
+func Test_LCOW_Layer_Integrity(t *testing.T) {
+	requireFeatures(t, featureLCOWIntegrity, featureLCOW)
+
+	client := newTestRuntimeClient(t)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	pullRequiredLCOWImages(t, []string{imageLcowK8sPause})
+
+	// Delete container image in case it already exists.
+	removeImages(t, []string{imageLcowAlpine})
+
+	// Pull image with dm-verity enabled.
+	pullRequiredLCOWImages(
+		t,
+		[]string{imageLcowAlpine},
+		WithSandboxAnnotations(map[string]string{
+			"containerd.io/diff/io.microsoft.storage.lcow.append-dm-verity": "true",
```

Comment:
This depends on [this](https://github.com/containerd/containerd/pull/6000) getting in as is right?

Reply:
yup, that too. added a feature flag to enable local testing at least.